README.md

```diff
@@ -1,9 +1,22 @@
-# jdd-landing-page
 
```

Comment:
Kita butuh menambahkan keterangan tentang apa yang harus dilakukan jika orang mau melakukan kontribusi, jadi kita butuh tambahan section tentang:

* Goals of this repo
* How to develop
* How to contribute

Reply:
@tegarimansyah akan membuat branch baru untuk menyelesaikan ini